File: CS/DXSample.Web/SchedulerDataModelCode/Appointments.cs
using System;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
namespace DXSample.Web.SchedulerDB {

    public partial class Appointments {
        public Appointments(Session session) : base(session) { }
        public override void AfterConstruction() { base.AfterConstruction(); }
    }

}

File: VB/DXSample.Web/SchedulerDataModelCode/Appointments.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Namespace DXSample.Web.SchedulerDB

	Partial Public Class Appointments
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		Public Overrides Sub AfterConstruction()
			MyBase.AfterConstruction()
		End Sub
	End Class

End Namespace
